assignments/objects.js

```diff
@@ -19,30 +19,76 @@ const example = {
 
 // Write your intern objects here:
 
-
-// ==== Challenge 2: Reading Object Data ==== 
+const Mitzi = {
+  id: 0,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+};
+
+const Kennan = {
+  id: 1,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M"
+};
+
+const Keven = {
+  id: 2,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+};
+
+const Gannie = {
+  id: 3,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "F"
+};
+
+const Antonietta = {
+  id: 4,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F"
+};
+console.log(Gannie);
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(Mitzi.name);
 // Kennan's ID
-
+console.log(Kennan.id);
 // Keven's email
-
+console.log(Keven.email);
 // Gannie's name
-
+console.log(Gannie.name);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(Antonietta.gender);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+function speak() {
+  return "Hello, my name is Kennan!"
+}
 
+console.log(speak());
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
+function multiplyNums(num1, num2) {
+
+    let product = num1 * num2;
+    return num1 * num2;
+
+   }
 
```

Comment:
same thing here with the Antonietta object, create a method on her named multiplyNums